content/recipes/hot-reload.md

```diff
@@ -167,3 +167,7 @@ $ npm run start:dev
 ```
 
 A working example is available [here](https://github.com/nestjs/nest/tree/master/sample/08-webpack).
+
+### TypeORM
+
+If you're using `@nestjs/typeorm`, you'll need to add `keepConnectionAlive: true` to your TypeORM configuration.
```

Comment:
Could we change it to a hint instead of defining another section? (using the `blockquote` tag)

Reply:
Where would the hint go? My thought process was that this is applicable to both with CLI and without CLI, so it shouldn't go in either section specifically. I could put it at the top, but since it's only for workflows involving TypeORM, it felt weird putting it next to "Webpack HMR overview" information

Reply:
@kamilmysliwiec